core/src/View/Kokkos_BasicView.hpp

```diff
@@ -385,23 +385,67 @@ class BasicView {
   }
 
  public:
+  // check for accessor_arg for managed view ctors
+  template <class... P>
+  explicit inline BasicView(
+      const Impl::ViewCtorProp<P...> &arg_prop,
+      std::enable_if_t<!Impl::ViewCtorProp<P...>::has_pointer &&
+                           !Impl::ViewCtorProp<P...>::has_accessor_arg,
+                       typename mdspan_type::mapping_type> const &arg_mapping)
+      : BasicView(arg_prop, arg_mapping, accessor_type{}) {}
+
+  template <class... P>
+
+  explicit inline BasicView(
```

Comment:
```suggestion
  template <class... P>
  explicit KOKKOS_FUNCTION BasicView(
```

This is the non-arg constructor right -- so it needs `KOKKOS_FUNCTION`?

Reply:
No: it's a ctor which allocates so it can't have KOKKOS_FUNCTION (!has_pointer).